src/main/java/com/synopsys/integration/alert/channel/jira/server/JiraServerChannelKey.java

```diff
@@ -32,4 +33,9 @@
     public String getUniversalKey() {
         return "channel_jira_server";
     }
+
+    @Override
+    public String getDisplayName() {
+        return JiraServerDescriptor.JIRA_LABEL;
```

Comment:
There's no circular dependency here? The descriptors use the keys.  Why not move the label constants into the key here and then use that anywhere JiraServerDescriptor.JIRA_LABEL.

Reply:
If it's static, it doesn't belong to an instance of the class, so I think this should be ok.